src/buza/views.py

```diff
@@ -13,40 +11,6 @@
 # Migrate to class based views
 
 
-def logged_out(request):
-    print('logged out')
-    logout(request)
-    # return redirect('logout')
-    return render(request, 'registration/logout.html', {'section': 'logged_out'})
-
-
-# Changing the user's password
-@login_required
```

Comment:
I am guessing the auth service will be using it's own login and out views

Reply:
Right, we can just use the `django.contrib.auth` implementations of these views for now, rather than having our own copies.